lib/api/v3/utilities/path_helper.rb

```diff
@@ -480,6 +480,10 @@ def self.work_package_relations(id)
             "#{work_package(id)}/relations"
           end
 
+          def self.work_package_file_links(id)
+            "#{work_package(id)}/file_links"
```

Comment:
why do we need this? We have the amendment of the `api_v3_paths` service in the `storages/engine.rb`.